k8s/charts/seaweedfs/values.yaml

```diff
@@ -30,7 +30,7 @@ global:
   # Z number of replica in other servers in the same rack
   replicationPlacment: "001"
   extraEnvironmentVars:
-    WEED_CLUSTER_DEFAULT: "sw"
+    WEED_CLUSTER_DEFAULT: "config"
```

Comment:
What no need to change the default value here?

Reply:
This is an artifact left over form debugging, my mistake.

Reply:
reverted to previous default in https://github.com/seaweedfs/seaweedfs/pull/5048/commits/601661c9e42feded4f73b9f320ec7dae185f92cb

We were debugging the interaction between `enableAuth`, `skipAuthSecretCreation`, and `existingConfigSecret` at the time and I mistaken believed the secret mount was related tot he `sw` default value when it was actually something else.